games/games.js

```diff
@@ -41,26 +41,30 @@ nameForm.addEventListener('submit', (e) => {
 
 teamOneAddButton.addEventListener('click', () => {
     // increment the current state for team one's score
-    
+    score1++;
     // display updated data in the current game div
+    teamOneLabel.textContent = score1;
 });
 
 teamTwoAddButton.addEventListener('click', () => {
     // increment the current state for team two's score
-
+    score2++;
     // display updated data in the current game div
+    teamTwoLabel.textContent = score2;
 });
 
 teamOneSubtractButton.addEventListener('click', () => {
     // decrement the current state for team one's score
-
+    score1--;
     // display updated data in the current game div
+    teamOneLabel.textContent = score1;
 });
 
 teamTwoSubtractButton.addEventListener('click', () => {
     // decrement the current state for team two's score
-
+    score2--;
     // display updated data in the current game div
+    teamTwoLabel.textContent = score2;
```

Comment:
you should be able to call displayCurrentGameEl() here instead of directly setting your textContent